src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -208,5 +208,35 @@ public void dropCourse(@PathVariable("enrollmentId") int enrollmentId) {
 
        // TODO
        // check that today is not after the dropDeadline for section
+       Enrollment enrollment = enrollmentRepository.findById(enrollmentId).orElse(null);
+       if(enrollment == null){
```

Comment:
The error checks here are stronger than other controllers. This should be implemented to the others later as well.